service_kit/logging/git.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+
+from . import logger
+
+GIT = "/usr/bin/git"
+
+
+def _pwd_is_git_repository() -> bool:
+    try:
+        subprocess.run(
+            [GIT, "rev-parse", "--is-inside-work-tree"],
+            check=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+
+        return True
+    except subprocess.CalledProcessError:
```

Comment:
Move the logging here and catch `FileNotFoundError` (log when git not found and when it's not a git repository)